js/validations/signup.js

```diff
@@ -28,13 +49,26 @@ function checkPasswordLength() {
     const passInput = document.getElementById("password");
     if (passInput.value.length < minimalLength) {
         passInput.classList.add("border-danger");
-        passWarning.innerText = "Password must contain at least " + minimalLength + " symbols."
+        passWarning.innerText = "Password must contain at least " + minimalLength + " symbols.";
     } else {
         passInput.classList.remove("border-danger");
         passWarning.innerText = "";
     }
 }
 
+function checkConfirmPassword(){
+    var password = document.getElementById('password');
+    var confirmPassword = document.getElementById('confirm-password'); 
+    var confirmPassWarning = document.getElementById('confirmPassWarning');

```

Comment:
First, we already use let.
Second, here is difference between them https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var